src/Plugins/jquery.calendar.js

```diff
@@ -2677,6 +2681,7 @@
                 dochange();
             },
             rf: function() {
+                option.eventItems = [];
```

Comment:
What is the use case for this assignment? It will remove all event caching. If you are addressing the duplicate event issue, then please check he `ConcatEvents` method to fix it.

Reply:
i have solved that issue just adding that line on the refresh function, but maybe it doesn't be the best place to put it, if it fails before i will check the ConcatEvents, i have checked but it doesn't solve that issue i don't remember exactly why... but, i belive that (ConcatEvents) it doesn't call every time that the calendar need to be refreshed.